src/it/scala/io/iohk/ethereum/ledger/BlockImporterItSpec.scala

```diff
@@ -153,6 +168,34 @@ class BlockImporterItSpec
     blockchain.getBestBlock().get shouldEqual newBlock3
   }
 
+  it should "return Unknown branch, don't start reorganisation(therefore no block/ommer validation) in case of PickedBlocks with block that has a parent who is not present in the chain(or ommer not in  chain)" in {
```

Comment:
can't this test description be simplified?

Reply:
return Unknown branch, in case of PickedBlocks with block that has a parent that's not in the chain?

Reply:
Sounds better yes :)